Childrens-Social-Care-CPD/Views/Shared/_ErrorLayout.cshtml

```diff
@@ -107,7 +107,18 @@
                     {
                         <li class="dfe-header__navigation-item" id="mmi-agencyResources">
                             <a class="dfe-header__navigation-link" href="/agency-resources">
-                                Agency Resources
+                                Agency resources
+                                <svg class="dfe-icon dfe-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" width="34" height="34">
+                                    <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+                                </svg>
+                            </a>
+                        </li>
+                    }
+                    @if (featuresConfig.IsEnabled(Features.PathwaysTraining))
+                    {
+                        <li class="dfe-header__navigation-item" id="mmi-pathwaysTraning">
+                            <a class="dfe-header__navigation-link" href="/pathways-social-work-leadership-modules-available">
```

Comment:
Query: Is this the right URL for the top-level navigation? Should it be the 'modules available' page or the 'all pathways overview' page.

Reply:
Checked with Dave - this link is incorrect and should take the user to the main Overview page. I'll correct it.